brambling/templates/brambling/event/organizer/custom_form_list.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'brambling/event/organizer/__base.html' %}
+
+{% load zenaida %}
+
+{% block title %}Forms – {{ block.super }}{% endblock %}
+
+{% block main %}
+	{{ block.super }}
+
+	{% include "brambling/event/_empty_shop_alert.html" %}
+
+	<p><a class="btn btn-primary" href="{% url 'brambling_form_create' event_slug=event.slug %}">Create form</a></p>
+	{% if custom_forms %}
+		<table class='table table-striped'>
+			<thead>
+				<tr>
+					<th>Name</th>
+					<th>Type</th>
+					<th>Index</th>
```

Comment:
I don't think users are going to understand what this field is. _I_ don't understand what this field is.